### ISSUE TITLE
Insert post splitter button only appear when fullscreen mode

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#264]
```
What's the problem?
Insert post splitter button only appear when fullscreen mode




What browser are you using?
Opera 11




What version of ScribeFire are you running?
ScribeFire 1.4.5

```

-----
Original issue reported on code.google.com by `fahmyro...@gmail.com` on 25 Feb 2011 at 2:59

**Comment by GoogleCodeExporter** [comment]
```
That probable means that your blog doesn't actually support the post splitter 
feature; it shouldn't be showing up in full-screen mode at all.

What kind of blog do you have?
```

-----
Original comment by `cfinke` on 25 Feb 2011 at 8:05
* Changed state: **Pending**

**Comment by GoogleCodeExporter** [comment]
```
I'm using blogger.
Actually I'm a new user, so I don't know whether blogger support post splitter 
or not.
But post splitter button appear here when using fullscreen mode, and I can 
click it too.
```

-----
Original comment by `fahmyro...@gmail.com` on 26 Feb 2011 at 6:56

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `cfinke` on 5 Mar 2011 at 9:35
* Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
I have the same bug with Opera, using Blogger. I know that my blog supports 
splitter because I can add it with Windows Live Writer and also by typing 
<!--more--> into the HTML mode of ScribeFire.
```

-----
Original comment by `arvis.la...@gmail.com` on 8 May 2011 at 2:41

**Comment by GoogleCodeExporter** [comment]
```
should be fixed in version 4.3.1+
```

-----
Original comment by `scribefi...@gmail.com` on 18 Jun 2014 at 2:19
* Changed state: **Fixed**
